js-exercises/sum-primes/sumPrimes.js

```diff
@@ -0,0 +1,38 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+function* getRange(start, end) {
+  let temp = start;
+  while (temp <= end) {
+    yield temp += 1;
+  }
+}
+
+const isQuotientZero = (dividend, divisor) => dividend % divisor === 0;
```

Comment:
Do you think `isDivisible` would be a better name here? Great work with this predicate.